### ISSUE TITLE
Deactivate WebUI/Api on fatpack recovery when off

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#4182]
see https://github.com/raspiblitz/raspiblitz/issues/4171#issuecomment-1728302628

**Comment by rootzoll** [comment]
Test once we have a first fatpack to test with.

**Comment by rootzoll** [comment]
OK will be part of v1.11.0rc5 closing.

**Comment by woiwoiwoiwoi** [comment]
So I seem to be the edge case you discovered in https://github.com/raspiblitz/raspiblitz/issues/4171#issuecomment-1728302628 and can't get the WebUI to work. It used to be deactivated in 1.8.0, but after the update to 1.10.0 fatpack (could not get minimal to work) I could access it, even though it was still deactivated in the menu. Then I activated it in the menu, but now it's broken. After putting in my password, I get:

`An error occurred: Unknown error. The response was: 502 Bad Gateway.`

I noticed that during install I get what you mentioned in https://github.com/raspiblitz/raspiblitz/issues/4171#issue-1894686808:

```
# WARNING! The given API repo is not available:
# user(fusion44) repo(blitz_api) branch(v1.10)
# WORKING WITH FALLBACK REPO - USE JUST FOR DEVELOPMENT - DONT USE IN PRODUCTION

# WARNING! The given WebUI repo is not available:
# user(raspiblitz) repo(raspiblitz-web) branch(v1.10)
# WORKING WITH FALLBACK REPO - USE JUST FOR DEVELOPMENT - DONT USE IN PRODUCTION
```

Is there a straight-forward way to fix this? Otherwise I'll just wait for 1.11.0 and keep using ssh.

**Comment by rootzoll** [comment]
Best to wait for v1.11.0 .. if you still have an error then please comment again here.
